torch/nn/modules/loss.py

```diff
@@ -780,6 +780,9 @@ class SmoothL1Loss(_Loss):
 
     beta is an optional parameter that defaults to 1.
 
+    Note: When beta is set to 0, this is equivalent to we call out directly to :class:`L1Loss`.
```

Comment:
this doesn't look grammatically correct, did you mix up two messages?  I would expect something like:

"Note: When beta is set to 0, this is equivalent to calling L1Loss."

Also, is that the correct note doc format?

Reply:
thanks, fixing :/ 

I skimmed a few other files containing docs to find the `:class:` anotation- but is there a command that I can run locally to test the formatting?